server/routerlicious/packages/lambdas/package.json

```diff
@@ -55,6 +55,7 @@
     "@fluidframework/protocol-definitions": "^0.1014.0",
     "@fluidframework/server-services-client": "^0.1014.0",
     "@fluidframework/server-services-core": "^0.1014.0",
+    "@fluidframework/server-services-utils": "^0.1014.0",
```

Comment:
lambdas are isomorphic (client bundle them as well). So this change reverts his PR. So you will probably need to keep both version alive.

Another way would be to pass validateToken as function to configureWebSocketServices() so the caller will be responsible to pass it, we only need to keep lambdas package isomorphic. Not the service caller. There is another relevant issue (https://github.com/microsoft/FluidFramework/issues/4010)